src/js/structs/PodContainer.js

```diff
@@ -30,7 +30,7 @@ module.exports = class PodContainer extends Item {
 
       default:
         return Object.assign(Object.create(PodContainerStatus.NA), {
-          displayName: StringUtil.capitalize(this.get('status').toLowerCase())
+          displayName: StringUtil.capitalize(this.getStatusString().toLowerCase())
```

Comment:
I aggree, however the `getStatusString` also returns a string even if the `status` is missing. If we use `get('status')` and `status` is not defined, this is going to raise an exception.